doc/raid.md

```diff
@@ -0,0 +1,73 @@
+# RAID setups - what works?
+
+## General considerations
+
+- How much stacking of multipath+raid+enc+lvm is allowed for /boot?
+- Avoid bcache for bootloader_partition and /boot?
+- general: / on full disk [raid] (i.e. - not a partition)?
+
+> All tests were run with tw-20180925 + libstorage-ng-4.1.37, yast2-storage-ng-4.1.19.
+
+## Terminology
+
+- raid1: raid level 1 with metadata format <= 1.0 (metadata at the end of the device)
+
+- raidX: any raid with any metadata format
+
+- bootloader_partition: partition containing the initial part of the bootloader (loaded by the system's firmware)
+    - x86, gpt: bios_grub partition
+    - x86, msdos: mbr gap
+    - efi (x86, aarch64): esp
+    - ppc64 (gpt, msdos): prep partition
+
+## Results
+
+- raid1 over full disk, partitioned, bootloader_partition on this disk
+    - non-efi: works, grub_installdevice contains N physical disks
+
+    - ! non-efi: grub2-install blocked in D-state for a very long time (3+ min)
+
+    - ! non-efi: pointing grub_installdevice to raid1 device directly results in error:
+      ```sh
+      grub2-install: error: diskfilter writes are not supported.
+      ```
+
+    - ! x86, msdos: wrong warning about missing bios_grub partition
+
+    - ! efi: (incorrect?) warning about esp on software raid
```

Comment:
well, it was discussed in respective feature. Problem is that we depends on firmware what it supports and how much firmware checks ESP partition. In general we do not want to guarantee that it works, but you should be able to accept warning.